tests/lib/rules/max-len.js

```diff
@@ -309,6 +314,17 @@ ruleTester.run("max-len", rule, {
                 }
             ]
         }, {
+            code: "// A comment that exceeds the max comment length, but not the specified max length for code.",
+            options: [40, 4, { comments: 80 }],
```

Comment:
i'm confused; code can be 40 but comments can be 80, but the comment falls between 80 and 40? (ordering of those numbers is intentional)